redocly.yaml

```
@@ -13,6 +13,15 @@ l10n:
       name: 日本語
 redirects:
   $ref: redirects.yaml
+responseHeaders: 
+  '**':
+    - name: X-Frame-Options
+      value: DENY
+    - name: frame-ancestors
+      value: NONE
```

Following up on the above, I think it's probably unnecessary to set `X-Frame-Options` (any browser that doesn't support `Content-Security-Policy` probably can't display the site properly regardless) but not harmful.

As for the frame ancestors, I think this is the right syntax:

```suggestion
    - name: Content-Security-Policy
      value: frame-ancestors 'none'
```